pageserver/src/tenant/timeline.rs

```diff
@@ -1175,22 +1179,34 @@ impl Timeline {
         let reconstruct_timer = crate::metrics::RECONSTRUCT_TIME
             .for_get_kind(get_kind)
             .start_timer();
```

Comment:
This metric and the get_reconstruct_values data metric are incorrectly counted right now. They no longer have clear semantics in a concurrent world.

What to do about them?

Reply:
fe615520ddf3c024f6ef896cd2197017280b1dd5